.ci/semaphore.yml

```diff
@@ -0,0 +1,37 @@
+version: v1.0
+name: "Checkstyle CI pipeline on Semaphore"
+agent:
+  machine:
+    type: e1-standard-2
+    os_image: ubuntu1804
+blocks:
+  - name: "Main"
+    task:
+      prologue:
+        commands:
+          - checkout
+          - cache restore m2
+          - eval "export M2_CACHE_SIZE=$(du -s $HOME/.m2 | cut -f1)"
+      epilogue:
+        commands:
+          - |
+            if [[ ${M2_CACHE_SIZE} -ne $(du -s $HOME/.m2 | cut -f1) ]]
+            then
+              cache delete m2
+              cache store m2 $HOME/.m2
+            fi
+      jobs:
+        - name: "Build"
+          matrix:
+            - env_var: JDK
+              values:
+                - "8"
+                - "11"
```

Comment:
can we do something different that we do not do on other CIs ?
or you plan todo this in separate PR?

Reply:
Yes, the idea is to remove some stages from the travis to speed up the whole process.

Reply:
can we use this CI to test other jdk builds ?
To remove item from Travis I need to trust this CI, so it is better to put on it something not critical and test it by time (1-2 month), github integrations, logs review, access to logs, ..... we need a lot of features.
Travis is most stable and quick CI, it host most critical items. You might saw try to move from Travis to Jenkins :) , that was failed as Jenkins never become stable and reliable ... .

Reply:
> can we use this CI to test other jdk builds ?

For other JDK builds we can use Github actions. The list of JDKs supported by Github own CI is here:
https://static.azul.com/zulu/bin/ 

> To remove item from Travis I need to trust this CI, so it is better to put on it something not critical and test it by time (1-2 month), github integrations, logs review, access to logs, ..... we need a lot of features.
> Travis is most stable and quick CI, it host most critical items. You might saw try to move from Travis to Jenkins :) , that was failed as Jenkins never become stable and reliable ... .

What if we move locale builds to this CI? test-de, test-fi and so on? This will speed up the Travis ci pipeline for 10 min at least.

Reply:
I suggest adding some tasks to this CI and wait for a while. If this will be stable for several months, then we can remove some jobs from Travis CI.